### ISSUE TITLE
Can the server reload config?

### ISSUE THREAD

**Issue by bhenderson** [code-mancers/invoker#201]
If I add commands to my invoker.ini file I have to restart the service. Can the daemon just reload the config anytime it needs it?
I was confused at the `add` command not working with newly added commands.
Would this feature be interesting?

**Comment by gnufied** [comment]
Yeah it is possible to implement that I think. The most trickiest part will be perhaps `foo.dev` dns must keep working, but it is possible.

**Comment by kgrz** [comment]
Probably related #11
